packages/myst-cli/package.json

```diff
@@ -53,6 +53,7 @@
   },
   "dependencies": {
     "@curvenote/blocks": "^1.5.17",
```

Comment:
Can we remove this now?

Reply:
💥 
![image](https://user-images.githubusercontent.com/913249/212794358-01a95050-b48e-44bc-9413-16f8703e2b81.png)